_includes/documentation-topics.html

```diff
@@ -0,0 +1,16 @@
+{% for item in site.data.pages %}
+    {% if item.title == "Documentation Topics" && item.subfolderitems %}
+    	<aside class="documentation-topics">
+    		<div class="grid-container grid-x grid-padding-x">
+		        <div class="cell small-12">
+		        	<h6>{{ item.title }}</h6>       
+		            <ul>
+		                {% for subitem in item.subfolderitems %}
+		                    <li><a href="{{ subitem.url | relative_url }}"{% if pg.target %} target="{{ pg.target }}"{% if pg.target == "_blank" %} rel="noreferrer noopener"{% endif %}{% endif %}>{{ subitem.title }}</a></li>
```

Comment:
This doesn't need to have the rel attributes if the plugin is handing that.

Reply:
Good catch.